### DISCUSSION TITLE
Format for annotation data

### DISCUSSION THREAD

**Discussion by Vortigern-The-Grey** [annotorious/annotorious#492] [General]
With the storing of annotations in a file, does it need to be json or js? Also, is there any specific formatting that needs to go around the annotation blocks?

**Comment by rsimon** [comment]
Hi @Vortigern-The-Grey,

I assume you are referring to files that you fetch directly via the `loadAnnotations` function? They need to be JSON. (The file name or extension are actually irrelevant. But the content needs to be valid JSON.)

The format must be identical to what you get out from `.getAnnotations`, which means a JSON array with objects adhering to the [Annotorious model](https://annotorious.dev/guides/data-model/) (or the W3C model in case you use the [W3C adapter](https://annotorious.dev/guides/data-model/#the-w3c-adapter)).

**Reply by Vortigern-The-Grey** [reply]
Thank you!
